scripts/sel4.sh

```diff
@@ -97,6 +101,12 @@ if [ "$DESKTOP_MACHINE" = "no" ] ; then
                     gcc-${compiler_version}-arm-linux-gnueabihf \
                     cpp-${compiler_version}-arm-linux-gnueabihf \
                     g++-${compiler_version}-arm-linux-gnueabihf \
+                    gcc-${compiler_version}-x86-64-linux-gnu \
```

Comment:
Should only create symlinks for the cross-compilers that exist.

Reply:
Also conditional now.